python/packages/ai/teams/ai/augmentations/__init__.py

```diff
@@ -7,10 +7,16 @@
 from .default_augmentation import DefaultAugmentation
 from .monologue_augmentation import MonologueAugmentation
 from .sequence_augmentation import SequenceAugmentation
+from .tools_augmentation import ToolsAugmentation
+from .tools_constants import ACTIONS_HISTORY, PARALLEL_TOOL_CALLS, TOOL_CHOICE
 
 __all__ = [
     "Augmentation",
     "DefaultAugmentation",
     "MonologueAugmentation",
     "SequenceAugmentation",
+    "ToolsAugmentation",
+    "ACTIONS_HISTORY",
```

Comment:
are these for internal use only or are we exposing them for devs? If they are internal only they should not be exported from the `__init__.py`